workers/unity/Packages/com.improbable.gdk.core/Worker/WorldCommandsReceivedStorage.cs

```diff
@@ -71,5 +83,114 @@ public void AddResponse(WorldCommands.EntityQuery.ReceivedResponse response)
         {
             return new ReceivedMessagesSpan<WorldCommands.EntityQuery.ReceivedResponse>(entityQueryResponses);
         }
+
+        ReceivedMessagesSpan<WorldCommands.CreateEntity.ReceivedResponse>
+            IDiffCommandResponseStorage<WorldCommands.CreateEntity.ReceivedResponse>.GetResponse(long requestId)
+        {
+            if (!createEntitySorted)
+            {
+                createEntityResponses.Sort(comparer);
+                createEntitySorted = true;
+            }
+
+            var responseIndex = createEntityResponses.GetResponseIndex(requestId);
+            if (responseIndex < 0)
+            {
+                return new ReceivedMessagesSpan<WorldCommands.CreateEntity.ReceivedResponse>(createEntityResponses, 0,
```

Comment:
Could we have an `ReceivedMessagesSpan<T>.Empty();`? Similar to the `Array<T>.Empty();`

Reply:
Done